### PR TITLE
Added Function plot_hist ready for review 

### PR DESCRIPTION
**Pull request by hfboyce**

### CODE REVIEW

File: colourblind8/colourblind8.py [hunk 1]
```diff
@@ -56,3 +56,46 @@ def plot_scatter(self, x, y, labels = None, palette = None,
 
         ax.legend(title = legend_title, bbox_to_anchor=(1.01, 1), loc = "upper left")
         return ax
+    
+    def plot_histogram(self, y, alpha = 1.0, labels = None, palette = None, 
+                title = None, x_lab = None, legend_title = None):
+
```
Comment (sabrinatkk) [inline colourblind8/colourblind8.py]: We are missing docstrings for `plot_histogram` here. I will add that in my pull request

File: setup.py [hunk 2]
```diff
@@ -5,14 +5,13 @@
     version='0.1dev',
     packages=['Colourblind8',],
     license='LICENSE.txt',
-    long_description=open('README.txt').read(),
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
      author = ['Ian Flores', 'Sabrina Tse', 'Hayley Boyce '],
     install_requires=[
         "numpy",
```
Comment (sabrinatkk) [inline setup.py]: Please double check if this code is correct and working.

File: README.md [hunk 3]
```diff
@@ -23,11 +23,14 @@ At this stage, `Colourblind8` will focus on developing settings for the three mo
...
+- `plot_lines()`:
+   inputs: x variable,  y variable(s),  labels = None, palette = None, 
+                title = None, x_lab = None, y_lab = None, legend_title = None
+   output: a matplotlib object
+  - This function implements a method that makes line graphs accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size as well as the face colour.
+- `plot_scatter(()`
   - This function implements a theme that makes plots accesible to people with protanopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
-- `set_trita()`
+- `plot_histogram()`
   - This function implements a theme that makes plots accesible to people with tritanopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
```
Comment (sabrinatkk) [inline README.md]: Hayley, please add the following to Colourblind8 Readme:

`help(Colourblind8)` for class docstring

`help(Colourblind8.modulename)` for module docstring

This is for users to view instructions on how to use our functions.  Thanks!

File: README.md [hunk 4]
```diff
@@ -23,11 +23,14 @@ At this stage, `Colourblind8` will focus on developing settings for the three mo
 
 ### Functions Included In This Package
 
-- `set_deutera()`:
-  - This function implements a theme that makes plots accesible to people with deuteranopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
-- `set_prota()`
+- `plot_lines()`:
+   inputs: x variable,  y variable(s),  labels = None, palette = None,
```
Comment (sabrinatkk) [inline README.md]: I will add alpha as a parameter to `plot_line()` and `plot_scatter` as well.

File: colourblind8/colourblind8.py (hunk 1 above)
Comment (hfboyce) [reply colourblind8/colourblind8.py]: Ok great thank you Sabrina

File: README.md (hunk 4 above)
Comment (hfboyce) [reply README.md]: I can add it to just plot_scatter. It doesn't make sense for alpha with a line plot.

File: README.md (hunk 3 above)
Comment (hfboyce) [reply README.md]: i added them to the readme!

File: README.md (hunk 4 above)
Comment (hfboyce) [reply README.md]: I added it to the scatter plot!

File: setup.py (hunk 2 above)
Comment (hfboyce) [reply setup.py]: This has been resolved

**Pull request closed without merging**